libs/designsystem/src/lib/components/item/item.component.scss

```diff
@@ -1,26 +1,33 @@
-@import '~@kirbydesign/core/src/scss/utils';
-@import 'item.utils';
+@use "sass:map";
+@use '~@kirbydesign/core/src/scss/utils';
+@use 'item.utils';
```

Comment:
No. The name of the module in line 3 is `item` and it's being using in line 55:

```scss
@include item.item-typography();
```

It _can_ be aliased to make it more obvious that it's different from `utils` that's being imported in line 2, e.g.:

```scss
@use 'item.utils' as local-utils;

// ...
// ...
// ...

@include local-utils.item-typography();
```

but it's not required since the modules have different names.